style.css

```diff
@@ -0,0 +1,338 @@
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.h4-connect {
+  width: 130px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.image {
+  width: 100%;
+  height: 220px;
+  top: 16px;
+  border-radius: 8px;
+  padding: auto;
+  align-items: center;
+}
+
+.card-works,
+.card-works1,
+.card-works2,
+.card-works3 {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  height: auto;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin: 6rem auto 0;
+}
+
+h2 {
+  text-align: left;
+  width: 290px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0;
+}
+
+.item-tag {
+  margin: 0 3rem 0 0;
+}
+
+.canopy {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+/* .icon-tag:not(:first-child) {
+  margin-left: 12px;
+} */
+
+button:not(:first-child) {
+  margin-left: 8px;
+}
+
+.langbutton button {
+  background: #ebebff;
+  border-radius: 8px;
+  color: #6070ff;
+  width: auto;
+  height: 24px;
+  font-size: 15px;
+  line-height: 20px;
+  letter-spacing: 0.03em;
+  border: none;
+  font-weight: 500;
+  padding: 0 0.5rem;
+}
+
+.bend {
+  padding: auto 10px;
+}
+
+.bend-yr {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  color: #7a869a;
+}
+
+.works {
+  padding: 16px;
+}
+
+.flex {
+  display: flex;
+}
+
+.item-center {
+  justify-items: left;
+  padding-left: 0;
+}
+
+.icon-tag:not(:first-child) {
+  margin: 0 0 0 16px;
+}
+
+ul,
+li {
+  list-style: none;
+  padding-left: 0;
+}
+
+a {
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+h1 {
+  font-size: 46px;
+  font-weight: 700;
+  line-height: 52px;
+  width: 327px;
+  padding-top: 50px;
+  margin-bottom: 0;
+  color: #172b4d;
+}
+
+.btn {
+  font-family: "Poppins", sans-serif;
+  width: 120px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 10px;
+  border: 1px solid #dfe1e6;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background: #fff;
+  margin: 12px 0;
+  color: #396df2;
+}
+
+.btn-big {
+  width: 140px;
+  height: 48px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 10px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.div {
+  margin-left: 0;
+}
+
+#aboutme {
+  border-top-right-radius: 80px;
+  margin-top: 8rem;
+  height: auto;
+  width: 100%;
+  padding: 16px;
+  background: #fff;
+  position: absolute;
+}
+
+/* .item-center {
+  align-items: center;
+} */
+
```

Comment:
- [ ] You should also delete these comments from your code for better practice and clean code, so that your code looks professional. :wink:  :smile:

Reply:
Done, thanks